mm2src/mm2_main/src/swap_versioning.rs

```diff
@@ -0,0 +1,26 @@
+//! Swap Versioning Module
+//!
+//! This module provides a dedicated type for handling swap versioning
+
+#[derive(Copy, Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct SwapVersion {
+    pub version: u32,
```

Comment:
Not a blocker but `u32` doesn't seem necessary for swap versioning, `u8` should be more than enough.

Reply:
I think it wont be a problem to move to u32, so we can start with u8
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2324/commits/bab63f1ac9ca5a32cc23462a093082847d7e8c7e